lib/fluent/plugin/out_forward.rb

```diff
@@ -456,6 +514,14 @@ def initialize(sender, server, failure:)
         @available = true
         @state = nil
 
+        # @hostname is used for certificate verification & TLS SNI
+        host_is_hostname = !(IPAddr.new(@host) rescue false)
+        @hostname = case
+                    when host_is_hostname then @host
+                    when @name then @name
```

Comment:
Yes, I changed the meaning of `name` in `<server>` section.
But in past versions, this `name` is used to indicate the role (or other human readable thing) of this server node in logging.
The "human readable role" is just FQDN server name signed by server certificate, isn't it?